osf/models/sanctions.py

```diff
@@ -689,13 +765,16 @@ def _on_complete(self, user):
             update_share(parent_registration)
 
     def approve_retraction(self, user, token):
-        self.approve(user, token)
+        '''Test function'''
+        self.approve(user=user, token=token)
 
     def disapprove_retraction(self, user, token):
-        self.reject(user, token)
+        '''Test function'''
+        self.reject(user=user, token=token)
```

Comment:
I would endorse just removing these and their Embargo counterparts from our codebase, but that's just extra work